### ISSUE TITLE
OLD bug: quest + summoned creatures

### ISSUE THREAD

**Issue by DimasjkTV** [suriyun-production/mmorpg-kit-docs#794]
The quest does not count if the monster was killed by your summoned creature.

**Comment by moepi2k** [comment]
I think therse also a problem that loot is locked when summoned creature kill something

**Comment by DimasjkTV** [comment]
With loot works well (single player)
[bandicam 2021-02-12 12-40-15-451.zip](https://github.com/suriyun-production/mmorpg-kit-docs/files/5970818/bandicam.2021-02-12.12-40-15-451.zip)

**Comment by insthync** [comment]
Fixed

**Comment by DimasjkTV** [comment]
> Fixed

Can you send the fix here? If it is in one script, please.

**Comment by insthync** [comment]
Okay, in `BaseMonsterCharacterEntity.cs` -> function `Killed`
add 
```
                        if (lastAttacker != null)
                            lastAttacker.TryGetEntity(out attackerCharacter);
```
to line 376

![Screenshot_207](https://user-images.githubusercontent.com/2697550/108022789-d13d8b00-7053-11eb-82d2-bbdd91fabfde.png)

**Comment by insthync** [comment]
Fixed in 1.62f
